packages/vm/src/testing/mock.rs

```diff
@@ -194,13 +204,14 @@ impl MockQuerier {
         let bin = match to_binary(request) {
             Ok(raw) => raw,
             Err(e) => {
-                let used_gas = e.to_string().len() as u64;
+                // FIXME: is this correct? Was this charged already?
```

Comment:
As above. MockQuerier will not charge anything, but the real one will charges external gas. So I guess doing this gives us more realistic tests